src/jquery.sceditor.js

```diff
@@ -69,8 +69,8 @@ define(function (require) {
 
 		options = options || {};
 
-		if (!options.runWithoutWysiwygSupport && !browser.isWysiwygSupported) {
-			return;
+		if (options.runWithoutWysiwygSupport || !browser.isWysiwygSupported) {
+			SCEditor.isWysiwygSupported = false;
```

Comment:
Oh, do you mean force source mode only regardless of WYSIWYG support?

The `runWithoutWysiwygSupport` option is to make the editor run when there is no browser WYSIWYG support. This is what it should do:

| WYSIWYG supported | Run without WYSIWYG |       Mode       |
|:-----------------:|:-------------------:|:----------------:|
|        Yes        |         Yes         |        Any       |
|        Yes        |          No         |        Any       |
|         No        |         Yes         | Source mode only |
|         No        |          No         |     Disabled     |

Forcing the editor to only run in source mode, regardless of if the browser supports WYSIWYG, isn't what `runWithoutWysiwygSupport` option is for. `runWithoutWysiwygSupport` is for what to do if there's no browser WYSIWYG support.